modules/admin-ui/src/main/webapp/scripts/modules/events/controllers/serieController.js

```diff
@@ -366,14 +378,15 @@ angular.module('adminNg.controllers')
     });
 
     $scope.themeSave = function () {
-        var theme = $scope.theme.current;
+        var selectedthemeid = $scope.selectedtheme.id;
+        $scope.updateSelectedThemeDescripton();
 
-        if (angular.isUndefined(theme) || theme === null) {
-            SeriesThemeResource.delete({id: $scope.resourceId}, {theme: theme}, function () {
+        if (angular.isUndefined(selectedthemeid) || selectedthemeid === null) {
```

Comment:
The JS-nerd in me wants to replace this with a simple `== null`, but I see why one would want to avoid explicit use of `==`. :disappointed: